adareducer/engine.py

```diff
@@ -228,6 +270,21 @@ def run(self):
             self.reduce_file(candidate)
             candidate = self.next_file_to_process()
 
+        # Try to brute-delete files again. This might give us new results when
+        # compared to the first time we called attempt_delete_all thanks to the
+        # project being reduced.
+        if DELETE_FILES_AFTER:
```

Comment:
I think this step can be removed completely: as part of self.reduce_file we're trying to delete each source already